graviton/README.md

```diff
@@ -1,9 +1,12 @@
 <!-- markdownlint-disable no-inline-html -->
 <!-- markdownlint-disable ol-prefix -->
+<!-- markdownlint-disable first-line-h1 -->
+<p align="center"><img  width=100%  src="https://infura-ipfs.io/ipfs/QmUJQFQET7VHyfDepM9CNhGBEabjH42XzK39kAisPWdE3K"  border="0" /></p>
 
-# GRAVITON (aka the TEAL Blackbox Toolkit): Program Reporting and Testing via Dry Runs
+Graviton (aka the TEAL Blackbox Toolkit): Program Reporting and Testing via Dry Runs
 
-<img width="345" alt="http://cds.cern.ch/record/2315186/files/scoap3-fulltext.pdf" src="https://user-images.githubusercontent.com/291133/160721859-21a3560a-0a82-4249-aa54-5ede4c60f8d2.png">
+<!-- Please let me know if you'd like to keep or you think the logo in the header is fine since it also encapsulates the diagram in the letter G -->
```

Comment:
```suggestion
<!-- Please let me know if you'd like to keep or you think the logo in the header is fine since it also encapsulates the diagram in the letter G -->
```
Right. It was a nice little link. I suggest you modify `CHANGELOG.md` and add something like

```
### Removed

* Original logo including _ALT_ link to graviton journal article in the following tag: `<img width="345" alt="http://cds.cern.ch/record/2315186/files/scoap3-fulltext.pdf" src="https://user-images.githubusercontent.com/291133/160721859-21a3560a-0a82-4249-aa54-5ede4c60f8d2.png">`
```

Reply:
@tzaffi sounds good, just pushed the change. Resolving, but let me know if there is anything else, otherwise ready to merge